controller/user-controller.js

```diff
@@ -0,0 +1,41 @@
+const userService = require('../services/user-services');
+
+const signUp = async (req, res) => {
+  try {
+    const {email, password, firstname, lastname, skintype} =req.body;
+
+    if (!email|| !password || !firstname || !lastname) {
+      throw new Error("KEY Error!");
+    }
+
+    await userService.signUp(email, password, firstname, lastname, skintype);
+    
+    return res.status(201).send({message: "SIGNUP_SUCCESS"});
+  } catch (err) {
+      res.status(err.statusCode || 400).json ({message: err.message});
+  }
+};
+
+const signIn = async (req, res) => {
+  try {
+    const { email, password }= req.body;
```

Comment:
마이너하지만 해당 부분은 지속 언급드린 사항과 동일합니다~!
코드 포맷 자체의 통일성을 위해서도 지속 확인해주세요
```suggestion
    const { email, password } = req.body;
```